posts/2022-01-14-construct-vendia-univseral-application-using-domain-driven-design.md

```diff
@@ -0,0 +1,177 @@
+# Construct Vendia Universal Application using Domain Driven Design
+
+Vendia Share is a data platform that you can use to quickly build real-time data sharing applications, which we call Universal Applications (or "Unis" for short). You can build both internal and external applications on Vendia Share. Internal applications include applications that you need for enterprise data management - transactional, analytical, and master data. External applications include applications for data exchange across trust boundaries between partners or a consortium. 
```

Comment:
```suggestion
Vendia Share is a platform that you can use to quickly build real-time data sharing applications, which we call Universal Applications (or "Unis" for short). 
```

Reply:
I don't believe the Internal vs. External distinction here is critical to include.  Going for brevity here given the good stuff that follows in later sections.